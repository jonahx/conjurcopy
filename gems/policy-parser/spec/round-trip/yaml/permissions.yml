- !permit
  resource: !variable foobar
  privilege: [ read, execute ]
  role:
  - !group developers
  - !group support
