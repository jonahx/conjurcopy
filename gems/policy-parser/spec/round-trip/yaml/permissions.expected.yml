---
- !permit
  privilege:
  - read
  - execute
  resource: !variable
    id: foobar
  role:
  - !group
    id: developers
  - !group
    id: support
